CONTRIBUTING.rst

```diff
@@ -135,14 +135,14 @@ More information regarding the usage of GitHub can be found in the `GitHub Guide
 Coding Guidelines
 -----------------
 
-Before your pull request can be merged into the codebase, it will be reviewed by one of the sim-pipeline developers and required to pass a number of automated checks. Below are a minimum set of guidelines for developers to follow:
+Before your pull request can be merged into the codebase, it will be reviewed by one of the slsim developers and required to pass a number of automated checks. Below are a minimum set of guidelines for developers to follow:
 
 General Guidelines
 ^^^^^^^^^^^^^^^^^^
 
-- sim-pipeline is compatible with Python>=3.7 (see `setup.cfg <https://github.com/herjy/sim-pipeline/blob/main/setup.cfg>`_). sim-pipeline *does not* support backwards compatibility with Python 2.x; `six`, `__future__` and `2to3` should not be used.
+- slsim is compatible with Python>=3.7 (see `setup.cfg <https://github.com/herjy/slsim/blob/main/setup.cfg>`_). slsim *does not* support backwards compatibility with Python 2.x; `six`, `__future__` and `2to3` should not be used.
```

Comment:
just noticed that the link points to a forked version

Reply:
Changed to correct link!